### ISSUE TITLE
Reproducing Plug-and-Play Results

### ISSUE THREAD

**Issue by mdaehl** [Xin-Ye-1/BEVDiffuser#6]
Hello, first of all, thank you for your great work and making it generally available.

I tried reproducing the results from Table 1 of your paper, where you demonstrate how integrating your method into the training of BEVFormer improves general detection performance. However, I am unable to reproduce these results.

My process was as follows:
I started by training the (tiny) Diffusion model according to your getting started guide and achieved even better results (mAP: 43.0 vs. 40.8 and NDS: 54.5 vs. 52.7 from Figure 4). Afterwards, I trained the plug-and-play model (DiffBEVFormer) using the config "diff_bevformer/layout_tiny.py" and the "tools/dist_train.sh" script. I had to halve the batch size to fit it on an A100 80GB (scaled the LR accordingly). Unfortunately, the results I am getting are not even close to the baseline.

When training the DiffBEVFormer, I noticed the loss is almost solely dominated by the BEV feature loss, as it is multiplied by 100. Reducing it to a factor of 10 helped, though the results were still drastically worse than the baseline. Is there anything I did wrong or missed?

Furthermore, do you plan to release the exact training settings or model weights for the base model? This would greatly help upcoming research!

Thanks in advance!

**Comment by Jasonkks** [comment]
@mdaehl Hi, did you manage to make it work for the plug-and-play model? Thanks.

**Comment by mdaehl** [comment]
@Jasonkks I was not able to reproduce the results for the tiny model. I achieved 37.4 NDS and 26.5 mAP. 
For the base model I was not able to reproduce the inital Diffusion model results from Figure 4, therefore I didn't train the Plug-and-play model.
